parlai/scripts/convo_render.py

```diff
@@ -0,0 +1,241 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import os
+import sys
+import json
+import random
+import argparse
+import tempfile
+
+# Constant Denoting the End of a Conversation
+END_OF_CONVO = "EOC"
+
+
+def gen_ul_style():
+    """ Generate general ul tag style"""
+    return (
+        "\t\tul{\n\t\t\tlist-style: none;\n\t\t\tmargin: 0;\n\t\t\tpadding: 0;\n\t\t}\n"
+    )
+
+
+def gen_ul_li_style():
+    """ Generate style for the li items for the conversations"""
+    return "\t\tul li{\n\t\t\tdisplay:inline-block;\n\t\t\tclear: both;\n\t\t\tpadding: 20px;\n\t\t\tborder-radius: 30px;\n\t\t\tmargin-bottom: 2px;\n\t\t\tfont-family: Helvetica, Arial, sans-serif;\n\t\t}\n"
```

Comment:
These would all be readable with
```python
"""
Multiline
Strings
"""
```

Reply:
Since these aren't dynamic, it also might be beneficial to just describe a large html template with all your styles:

HTML_TEMPLATE = """
<html>
<head>
   <style>...all your CSS classes</style>
</head>
...
"""
You can look at `interactive_web.py` for an example of this.

Reply:
I was hoping to keep it modular so that later if something small needs to be changed, one knows where to change it, but definitely using the f-strings and multiline suggestion you gave, thanks for that!

Reply:
I'd rather cross that bridge when we come to it. It's unlikely we'll need to customize output in a way that would be a one-off-and-throw-away modification.

Reply:
Sure I'll change it in my next commit

Reply:
Done in the latest commit